crypto-common/src/lib.rs

```diff
@@ -12,6 +12,9 @@
 #[cfg(feature = "std")]
 extern crate std;
 
+/// Hazardous materials.
```

Comment:
I put this here for now to avoid modifying the now `hazmat.rs` but it would be good to move the comment into there in a followup commit